src/OddDotNet/Span.cs

```diff
@@ -0,0 +1,73 @@
+namespace OddDotNet;
+
+public class Span
+{
+    public required Scope Scope { get; set; }
+    public required string Name { get; set; }
+    public required Dictionary<string, object> Attributes { get; set; }
+    public List<SpanEvent>? Events { get; set; }
```

Comment:
All of these `Collection<T>?` should be `Collection<T>`. The collection should be initialized, and only a getter should be provided, no setter.

Reply:
Done